epub33/rs/index.html

```diff
@@ -364,10 +364,11 @@ <h4>Base Direction</h4>
 				<h3>Unique Identifier</h3>
 
 				<p id="confreq-rs-pkg-unique-id" data-tests="#confreq-rs-pkg-unique-id">Reading Systems SHOULD NOT
-					depend on the <code>unique-identifier</code> attribute being unique to one and only one EPUB
-					Publication. Determining whether two EPUB Publications with the same Unique Identifier represent
-					different versions of the same publication, or different publications, may require inspecting other
-					metadata, such as the last modification date, titles, and authors.</p>
+					depend on the <a href="https://www.w3.org/TR/epub-33/#dfn-unique-identifier">Unique Identifier</a>
```

Comment:
Here or somewhere else, could we link to https://www.w3.org/TR/epub-33/#sec-opf-dcidentifier? That section is much more explicit about the fact that the unique-identifier is not actually the Unique Identifier itself :)

Reply:
I've updated the definition for Unique Identifier to be more specific that it is the value of the dc:identifier specified by the unique-identifier attribute (and added linking). Let me know if this works better for you now.

Reply:
Much better :)